Formula/ike-scan.rb

```diff
@@ -3,7 +3,7 @@ class IkeScan < Formula
   homepage "https://github.com/royhills/ike-scan"
   url "https://github.com/royhills/ike-scan/archive/1.9.5.tar.gz"
   sha256 "5152bf06ac82d0cadffb93a010ffb6bca7efd35ea169ca7539cf2860ce2b263f"
-  license "GPL-3.0-or-later"
+  license "GPL-3.0-or-later" => { with: "openvpn-openssl-exception" }
```

Comment:
Exception added to source code, e.g. https://github.com/royhills/ike-scan/blob/1.9.5/ike-scan.c#L20-L29
```
 * In addition, as a special exception, the copyright holders give
 * permission to link the code of portions of this program with the
 * OpenSSL library, and distribute linked combinations including the two.
 *
 * You must obey the GNU General Public License in all respects
 * for all of the code used other than OpenSSL.  If you modify
 * file(s) with this exception, you may extend this exception to your
 * version of the file(s), but you are not obligated to do so.  If you
 * do not wish to do so, delete this exception statement from your
 * version.
```

It was added in v1.8 according to NEWS https://github.com/royhills/ike-scan/blob/1.9.5/NEWS#L185-L187
```
* Added OpenSSL exception to the copyright notice, which permits linking of the
  program against OpenSSL.  This is to allow the use of OpenSSL in this GPL v2
  licensed program.
```